lib/jxl/enc_gaborish.cc

```diff
@@ -20,9 +20,9 @@ void GaborishInverse(Image3F* in_out, float mul, ThreadPool* pool) {
 
   // Only an approximation. One or even two 3x3, and rank-1 (separable) 5x5
   // are insufficient.
```

Comment:
Please update this comment. If I understand correctly, these weights are not only an approximation of the inverse of the 3x3 decoder Gaborish. I'm not sure what they actually represent.

Reply:
They are and always will be an approximation. A full exact transform would be infinite in size and only work for infinite-sized images.

Reply:
Of course, but within what can be done in a 5x5 convolution, one set of weights will be closer to the inverse of the decoder convolution than another. If it's intentionally not trying to be as close as possible to the inverse of the 3x3, that's fine, but then I wouldn't pretend that it's an approximation that is only a bit imprecise due to the limited kernel size.